api/net/Areas/Editor/Controllers/StorageController.cs

```diff
@@ -196,5 +197,174 @@ public IActionResult Delete(string path, [FromRoute] string? location = "capture
         System.IO.File.Delete(safePath);
         return new JsonResult(info);
     }
+
+    /// <summary>
+    /// Make a clip from the target file identified in the clip parameter.
+    /// </summary>
+    /// <param name="fileName"></param>
+    /// <param name="directory"></param>
+    /// <param name="start"></param>
+    /// <param name="end"></param>
+    /// <param name="clipNbr"></param>
+    /// <returns></returns>
+    [HttpGet("clip")]
+    [Produces("application/json")]
+    [ProducesResponseType(typeof(FolderModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "Storage" })]
+    public IActionResult CreateClip(string fileName, string directory, int start, int end, int clipNbr, string target)
```

Comment:
This probably should be an async function.

Reply:
As I'm waiting for the clip process to complete so I can return a reference to the newly created file, and be sure the file does exist, I figured this should be synchronous. Does async make sense in this case?

Reply:
You can wait an async function.  The point of async isn't always to fire and forget, but to not block.

Reply:
Fixed...and the name of the "target" parameter has been changed to "prefix" throughout.